__tests__/app.test.js

```diff
@@ -159,6 +159,44 @@ describe('/api/articles/:article_id', () => {
 });
 
 
+describe('/api/articles/:article_id/comments', () => {
+    test('GET:200 responds with an array of comment objects with required properties', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
+                expect(comments).toEqual(expect.any(Array));
+                expect(comments[0]).toEqual(
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                    })
+                );
+            });
+    });
+    test('GET:400 sends appropriate status and error message when given an invalid id', () => {
+        return request(app)
+            .get('/api/articles/not-an-article/comments')
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe('Invalid ID');
+            });
+    });
+    test('GET:404 sends appropriate status and error message when given a valid but non-existent id', () => {
+        return request(app)
+            .get('/api/articles/3141/comments')
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe('Article does not exist')
+            })
+    });
+});
```

Comment:
I'd also check for a article which has no comments associated with it (should still be 200 OK)

Reply:
Hey, just wondering how I test for that if all the articles have comments?